test/SILOptimizer/escape_effects.sil

```diff
@@ -414,3 +414,12 @@ bb0(%0 : $*Wrapper):
   %2 = struct_element_addr %0, #Wrapper._k
   return %2
 }
+
+sil [ossa] @call_mutate_accessor : $@convention(thin) (@inout Wrapper, @owned Klass) -> () {
```

Comment:
The CHECK lines are missing here.
(Note that the main test file for EscapeUtils is SILOptimizer/escape_info.sil. But testing the change here is also fine).